php/hireinsocial/src/HireInSocial/Infrastructure/InMemory/InMemoryThrottle.php

```diff
@@ -12,7 +12,7 @@ final class InMemoryThrottle implements Throttle
 
     public function isThrottled(string $id): bool
     {
-        return \array_key_exists($id, $this->throttles);
+        return isset($this->throttles[$id]);
```

Comment:
this is really faster than `array_key_exists` ? I wasn't even aware of that, are there any benchmarks I could check?

Reply:
It's a liiiiiiitle faster. In my opinion it's better to use language constructions instead of native functions. Up to you. :)

Reply:
Yes, but this not detect null https://stackoverflow.com/a/3210982

Reply:
@andrewprofile zmienię na polski. Wiesz co, generalnie to `isset()` sprawdza czy wartość istnieje w tablicy i czy ma jakąkolwiek wartość lub `null`. `empty()` sprawdza czy tablica ma klucz oraz wartość. Wyklucza też `null`. `array_key_exists()` sprawdza jedynie czy klucz istnieje w tablicy. `array_key_existst()` jako jedyny z trójki jest funkcją językową a nie jego konstrukcją.

Reply:
@patrykwozinski Yes `isset()` contains `array_key_exists ()`, but if `isset()` returns true then `array_key_exists()` will not be called, only if false returns it is checked if this key exists.

Reply:
yeah I don't see how `null` could be a valid value for configuration but who knows what future will bring us, I think at this point I would like then stick to `array_key_exists`. Sounds good?